lectures/commod_price.md

```diff
@@ -396,6 +394,7 @@ while error > tol:
 ax.plot(grid, price, 'k-', alpha=0.5, lw=2, label=r'$p^*$')
 ax.legend()
 ax.set_xlabel('$x$', fontsize=12)
```

Comment:
@SylviaZhaooo would you mind to remove the `fontsize` here (unless it is required for some reason).

Reply:
I have removed the 'fontsize'. Please let me know if any issues arise or if further adjustments are needed.